BedrockServer.cpp

```diff
@@ -1521,7 +1527,10 @@ void BedrockServer::postPoll(fd_map& fdm, uint64_t& nextActivity) {
     if (_shutdownState.load() == START_SHUTDOWN) {
         if (!_lastChance) {
             _lastChance = STimeNow() + 5 * 1'000'000; // 5 seconds from now.
```

Comment:
Now that we're using `_shutdownSet`, do we still need to check `_lastChance` here?

Reply:
Not really but I think the other PR can remove it.